src/GraphQL/Introspection/__Type.cs

```diff
@@ -27,7 +27,7 @@ public __Type(bool allowAppliedDirectives = false)
         /// </param>
         public __Type(bool allowAppliedDirectives = false, bool deprecationOfInputValues = false)
         {
-            Name = nameof(__Type);
+            SetName(nameof(__Type), validate: false);
```

Comment:
These changes align all of the introspection types to set their own type name in the same fashion.  (Some other existing introspection types already set the name this way.)  Previously this code was not necessary as the default name matched the set name and so skipped validation anyway.